kubernetes/istio/istio/src/test/resources/route-rule-reviews-test-v2.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: config.istio.io/v1alpha2
```

Comment:
It's not about this file. but I can see empty file [kubernetes/istio/istio/src/test/resources/istiotest.txt ](https://github.com/arquillian/arquillian-cube/pull/962/files#diff-d1d4386f1b71134234549ddf70ffd471)

Reply:
Yes it is normal, to check that the regular expression works as expected.